dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/resources/OpenAPI.nlsprops

```diff
@@ -17,10 +17,46 @@
\ No newline at end of file
+OPENAPI_DOCUMENT_VALIDATION_WARNING.useraction=Provide a valid OpenAPI document that resolves the reported warning.
+
+CSS_NOT_PROCESSED=CWWKO1652W: The server cannot read the specified CSS document {0} due to {1} : {2}.
+CSS_NOT_PROCESSED.explanation=The CSS document cannot be processed due to a valid error.
+CSS_NOT_PROCESSED.useraction=Provide a valid CSS document.
+
+UNSUPPORTED_CSS_VALUE=CWWKO1653W: The value that was specified for the {0} property is not supported. The value must be {1}.
+UNSUPPORTED_CSS_VALUE.explanation=The value of the specified property is either invalid or is not supported by the OpenAPI framework.
+UNSUPPORTED_CSS_VALUE.useraction=Provide a valid value for the property within the custom CSS file for the OpenAPI UI. 
+
+INVALID_CSS_BACKGROUND_IMAGE=CWWKO1654W: The background image that was specified at {0} either does not exist or is invalid.
+INVALID_CSS_BACKGROUND_IMAGE.explanation=The background image either does not exist or is invalid.
+INVALID_CSS_BACKGROUND_IMAGE.useraction=Provide a valid image.
+
+CUSTOM_CSS_NOT_PROCESSED=CWWKO1655W: The custom CSS file {0} that was specified for the OpenAPI UI was not processed. The server will restore the default values for the OpenAPI UI. Reason={1} : {2}.
+CUSTOM_CSS_NOT_PROCESSED.explanation=The server cannot process the custom CSS file. The server will restore the default values for the OpenAPI UI.
+CUSTOM_CSS_NOT_PROCESSED.useraction=Provide a valid custom CSS file for the OpenAPI UI along with all the dependencies.
+
+CSS_SECTION_NOT_FOUND=CWWKO1656W: The server read the specified CSS document {0}, but was unable to find a [.swagger-ui .headerbar ].
+CSS_SECTION_NOT_FOUND.explanation=The CSS document that is specified does not contain [.swagger-ui .headerbar ], or it is not formatted to meet the constraints that are required by the server.
+CSS_SECTION_NOT_FOUND.useraction=Provide a valid CSS file that contains a [.swagger-ui .headerbar ] in the required format.
+
+OPENAPI_FILTER_LOAD_ERROR=CWWKO1657E: Failed to load OpenAPI filter class: {0}.
+OPENAPI_FILTER_LOAD_ERROR.explanation=Specified OpenAPI filter class could not be found or loaded.
+OPENAPI_FILTER_LOAD_ERROR.useraction=Make sure the class specified exists and implements OASFilter interface.
+
+OPENAPI_MODEL_READER_LOAD_ERROR=CWWKO1658E: Failed to load OpenAPI model reader class: {0}.
+OPENAPI_MODEL_READER_LOAD_ERROR.explanation=Specified OpenAPI model reader class could not be found or loaded.
+OPENAPI_MODEL_READER_LOAD_ERROR.useraction=Make sure the class specified exists and implements OASModelReader interface.
+
+OPENAPI_FILE_PARSE_ERROR=CWWKO1659E: Failed to parse OpenAPI document for application: {0}.
+OPENAPI_FILE_PARSE_ERROR.explanation=Failed to parse OpenAPI document from META-INF folder.
+OPENAPI_FILE_PARSE_ERROR.useraction=Make sure the file is in correct YAML or JSON format.
+
```

Comment:
Make sure _that_ the file

Reply:
ok, good catch, same for above messages 'Make sure <i>that</i> the class'?